### ISSUE TITLE
Devtools: Saving a CPU profile produces empty file

### ISSUE THREAD

**Issue by bpasero** [electron/electron#7403]
* Electron version: 1.3.7
* Operating system: Mac OS and Windows 10

Steps:
* run a CPU profile from dev tools
* stop it (you get some output)
* save the file

=> the resulting file has 0KB

I cannot reproduce in Chrome 53, but I can reproduce in a bare electron app.

**Comment by kevinsawicki** [comment]
I can also reproduce this in 1.4.1

**Comment by deepak1556** [comment]
Profile tab seems to pass empty contents for `InspectorFrontendHostImpl.save` , while the following works fine
`require('electron').remote.getCurrentWindow().devToolsWebContents.executeJavaScript("InspectorFrontendHost.save('hello.txt', 'test', true)")` . Have to verify if its a upstream issue that was fixed.

**Comment by kevinsawicki** [comment]
I'm seeing the following error log when trying to save, not sure if it is related:

```
Failed to clear temp storage: It was determined that certain files are unsafe for access within a Web application, or that too many calls are being made on file resources.
```

**Comment by deepak1556** [comment]
Thanks @kevinsawicki that narrows the issue, this is a regression introduced by https://github.com/electron/electron/pull/6952 that made filesystem access available for custom standard schemes but not for devtools scheme, will fix it now.

**Comment by bpasero** [comment]
Thanks for fixing this!

**Comment by bronson** [comment]
If anyone else runs into this, it should be fixed in Electron 1.4.2.

Atom 1.16 is on Electron 1.3.13, and 1.17 is on 1.3.14...  Both produce 0-length CPU profiles.

Next Electron bump should take care of it.
